tests/page-layout-both/OPS/package.opf

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Fixed Layout</dc:coverage>
+    <dc:creator>Wendy Reid</dc:creator>
+    <dc:date>2021-11-24</dc:date>
+    <dc:description>If a reflowable spine item follows a pre-paginated one, it should appear on the next page.</dc:description>
+    <dc:identifier id="pub-id">page-layout-both</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>page-layout-both</dc:title>
```

Comment:
IMO this should not be called page-layout-both (title or ID). I realize that https://www.w3.org/TR/epub-rs-33/#page-layout-both is a pre-existing anchor, but it looks like a mistake — there is nothing in this package file about page-layout. I only see rendition:layout-reflowable in the spine, but that spec reference is to the rendition:page-spread-* section (which doesn't appear to be set in this test). Is something missing? Regardless, can you find a more appropriate name (or explain why this is a good name)?

Reply:
Still at a loss for the name on this one and I don't think it's a huge priority, so I vote to leave it as is.